utils/strings/validations.ts

```diff
@@ -0,0 +1,3 @@
+export function isEmpty(value: string) {
+  return !value || !value.trim();
```

Comment:
Just now we realised sending strings with only whitespaces through postman passes this check. It fails for console. I guess we need to figure out why.